### DISCUSSION TITLE
Copy/apply changes to a vehicle to multiple vehicles

### DISCUSSION THREAD

**Discussion by Ivan-Velickovic** [voxeltycoon/issues#346] [General]
This would mainly be to avoid the repetitiveness of having to apply a certain change to every single vehicle individually.

For example, say I want to upgrade all the trucks on a route to ones with more capacity, in order to this currently (as far as I know), I would have to send all the trucks to a garage and then upgrade all of them individually. The faster way to do it would be change one of the trucks, sell all the other ones, then copy the one I changed. I feel like it would be faster if I could just apply the changes of one truck to multiple as quickly as copying it.

The same can be applied to when I want to change many trucks to transport a different resource.

**Comment by andrewpey** [comment]
This is on our [roadmap](https://trello.com/b/wHz5JHqH/voxel-tycoon-roadmap) at Stage 1:
![image](https://user-images.githubusercontent.com/1748681/85923032-5f2ff000-b890-11ea-8f26-4ae2eb7c0c0b.png)

**Comment by Ivan-Velickovic** [comment]
My bad I didn’t see it on the roadmap, thanks!
